src/python/pants/option/global_options.py

```diff
@@ -659,7 +636,7 @@ def register_bootstrap_options(cls, register):
             type=bool,
             advanced=True,
             default=False,
-            removal_version="1.30.0.dev0",
+            removal_version="1.31.0.dev0",
```

Comment:
I decided that it was better to be conservative here, and not delete this code until pantsd has been on by default for a version. Looking forward to removing it though, as it's about 1k lines and thirdparty dep!